README.md

```diff
@@ -38,6 +38,93 @@ deactivate
 ```
 
 
+# Configuration
+
+
+## Main configuration file
+
+The main configuration file uses the TOML format. The default file name is `.pyodk_config.toml`, and the default location is the user home directory. The file name and location can be customised by setting the environment variable `PYODK_CONFIG_FILE` to some other file path. The expected file structure is as follows:
+
+```
+[central]
+base_url = "https://www.example.com"
+username = "my_user"
+password = "my_password"
+default_project_id = 123
+```
+
+
+## Session cache file
+
+The session cache file uses the TOML format. THe default file name is `.pyodk_cache.toml`, and the default location is the user home directory. The file name and location can be customised by setting the environment variable `PYODK_CACHE_FILE` to some other file path. This file should not be pre-configured as it is used to store a session token after login.
```

Comment:
THe -> the

I was a little surprised to see the home directory as the default rather than the working directory. Maybe getting working directory is problematic? Or maybe you're making the (totally valid) assumption that most people have a single Central server to work with.

Reply:
The config file contains a password, and the cache file contains an auth token which is like a password, and the home directory is usually private to the user. I assumed the general use case was a data manager with one instance and one or more projects. For more complex use cases these can be customised via the environment variables, or now by passing init params to `Client`:

```python
# This
os.environ["PYODK_CONFIG_FILE"] = ".my_config.toml"
os.environ["PYODK_CACHE_FILE"] = ".my_cache.toml"
with Client() as client:
    projects = client.projects.read_all()

# or this
with Client(config_path="my_config.toml", cache_path="my_cache.toml") as client:
    projects = client.projects.read_all()
```